Mathlib/MeasureTheory/Measure/AddContent.lean

```diff
@@ -73,28 +73,24 @@ instance : FunLike (AddContent C) (Set α) (fun _ ↦ ℝ≥0∞) where
 
 namespace AddContent
 
-@[ext] protected lemma ext {m m' : AddContent C} (h : ∀ s, m s = m' s) : m = m' := by
-  refine FunLike.coe_injective ?_
-  ext s
-  exact h s
+@[ext] protected lemma ext {m m' : AddContent C} (h : ∀ s, m s = m' s) : m = m' := FunLike.ext _ _ h
 
-protected lemma ext_iff (m m' : AddContent C) : m = m' ↔ ∀ s, m s = m' s :=
-  ⟨fun h s ↦ by rw [h], fun h ↦ AddContent.ext h⟩
+protected lemma ext_iff (m m' : AddContent C) : m = m' ↔ ∀ s, m s = m' s := FunLike.ext_iff
 
-@[simp] protected lemma empty {m : AddContent C} : m ∅ = 0 := m.empty'
+@[simp] protected lemma apply_empty {m : AddContent C} : m ∅ = 0 := m.empty'
 
-protected lemma add (m : AddContent C) (h_ss : ↑I ⊆ C)
+protected lemma sUnion_eq_sum (m : AddContent C) (h_ss : ↑I ⊆ C)
```

Comment:
Okay this is a possibly even more confusing name now. It sounds like you're saying that a union is equal to a sum, but of course what you're really saying is that the **image** of a union is equal to a sum (of the images). I think you won't get away with avoiding `apply` or `meas` in the name.

Reply:
I don't really want to prefix all names in the file with `apply`, though. How about I just call that `addContent.sUnion`?

Reply:
How about I name these lemma the same way it is done for measures: instead of `AddContent.sUnion`, I get it out of the namespace and call it `addContent_sUnion`.

Reply:
That's definitely an improvement, but do know that lemmas like `m (a ∩ b) + m (a ∪ b) = m a + m b` will need to be called `addContent_inter_add_addContent_union` (which I think is alright)

Reply:
If I was naming something as ubiquitous as a measure that would indeed be an issue, but this addContent should not see much use. A few long names is fine.